src/plugins/gc-subway/index-en.hbs

```diff
@@ -0,0 +1,105 @@
+---
+{
+	"title": "GC Subway Index page",
+	"language": "en",
+	"altLangPrefix": "index",
+	"parentdir": "gc-subway",
+	"pageclass": "cnt-wdth-lmtd",
+	"noContentTitle": true,
+	"breadcrumb": [
+	{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" }
+	],
+	"dateModified": "2020-12-04",
+	"share": "true"
+}
+---
+
+{{>alert-provisional-demos}}
+
+<h1 property="name" id="wb-cont" class="provisional gc-thickline">GC Subway Index</h1>
+
+<section class="provisional alert alert-info">
+	<h2 class="h3">This is an optional Alert</h2>
+	<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+		ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+		ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+		reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
+		sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+		est laborum.</p>
+</section>
```

Comment:
Please remove the alert. We wouldn't want to convey that this is a template. The example should stick to the component as much as possible